assets/airports/eddf.json

```diff
@@ -450,11 +528,11 @@
       "entryPoints": {
         "T173": ["GAPLA", ["DEMAB", "A160-"], ["SOLVU", "A130-"]]
```

Comment:
Additionally, all the entries under `arrivals` will need to be amended, since they contain an invalid requested routing of `T180.UNOKO25S.EDDF` / `T150.ROLIS25N.EDDF` / `T173.KERAX25N.EDDF` / `T163.PSA25S.EDDF`.